tests/unit/test_awsrequest.py

```diff
@@ -20,7 +20,10 @@
 import socket
 import sys
 
-from mock import Mock, patch
+try:
+    from mock import Mock, patch
```

Comment:
I think you've already added this in test. Let's consolidate this to `from test.mock import Mock, patch`.

Reply:
Changed in commit 014a4657.